### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -874,72 +874,72 @@ msgstr "не вдалося записати параметри до конфі
...
 #, c-format
 msgid "destination path '%s' already exists and is not an empty directory."
-msgstr ""
+msgstr "шлях призначення '%s' вже існує і не є порожньою директорією."
 
 #: builtin/clone.c:977
 #, c-format
 msgid "repository path '%s' already exists and is not an empty directory."
-msgstr ""
+msgstr "шлях до репозиторія '%s' вже існує і не є порожньою директорією"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "шлях до репозиторія '%s' вже існує і не є порожньою директорією."
```

File: po/uk.po [hunk 2]
```diff
@@ -874,72 +874,72 @@ msgstr "не вдалося записати параметри до конфі
...
 #: builtin/clone.c:1033
 #, c-format
 msgid "Cloning into '%s'...\n"
-msgstr ""
+msgstr "Клонування в '%s'...\n"
 
 #: builtin/clone.c:1062
 msgid "clone --recursive is not compatible with both --reference and --reference-if-able"
-msgstr ""
+msgstr "clone --recursive не сумісне з --reference та --reference-if-able"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "clone --recursive не сумісне з --reference та --reference-if-able одночасно"
```

File: po/uk.po [hunk 3]
```diff
@@ -2143,7 +2146,7 @@ msgstr "  (використовуйте \"git commit\" для завершенн
 
 #: wt-status.c:1244
 msgid "You are in the middle of an am session."
-msgstr ""
+msgstr "Ви в середені am сеансу."
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "Ви всередині am сеансу."
```

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: Good catch!

**Pull request merged**